bsp4j/src/main/java/ch/epfl/scala/bsp4j/JvmExtension.xtend

```diff
@@ -31,13 +32,24 @@ class JvmTestEnvironmentParams {
   }
 }
 
+@JsonRpcData
+class JvmMainClass {
+  @NonNull @SerializedName("class") String className
+  @NonNull List<String> arguments
+    new(@NonNull String className, @NonNull List<String> arguments) {
+    this.className = className
+    this.arguments = arguments
```

Comment:
Shouldn't the arguments be only defined when actually running the main? Why do we need it here?